### ISSUE TITLE
conf.level does not change calculations forprop.test

### ISSUE THREAD

**Issue by Lakens** [sdorairaj/binom#1]
Hi, if I am correct, the prop.test CI is not updated by setting the parameter conf.level. A working example below. The first 2 lines yield the same CI, but the should yield the same as the last two. It works fine for "ac". Hope this feedback helps (and it is not my own error).

binom.confint(5, n = 100, method = "prop.test", conf.level = 0.95)
binom.confint(5, n = 100, method = "prop.test", conf.level = 0.9)

binom.confint(5, n = 100, method = "ac", conf.level = 0.95)
binom.confint(5, n = 100, method = "ac", conf.level = 0.9)

prop.test(x = 5, n = 100, conf.level = 0.95)$conf.int
prop.test(x = 5, n = 100, conf.level = 0.9)$conf.int

**Comment by DanChaltiel** [comment]
Indeed, you can see at https://github.com/cran/binom/blob/c79bce79608360a7b617af3362cf824d082f6878/R/binom.confint.R#L141 that `prop.test()` is called without setting `conf.level`.

@sdorairaj If the code was published here, I would be glad to make a pull request 👍 (there is no point making a PR on CRAN's mirror)
